app/src/main/res/layout-land/item_movie.xml

```diff
@@ -5,6 +5,18 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
+    <View
+        android:id="@+id/rectangle_at_the_top"
+        android:layout_width="match_parent"
+        android:layout_height="10dp"
+        android:layout_alignTop="@+id/tvTitle"
+        android:layout_alignStart="@+id/tvOverview"
+        android:layout_marginTop="20dp"
+        android:background="#4d4e4f"
```

Comment:
you should name this color & add it to your resource in case you want to reuse it in the future